Makefile

```diff
@@ -262,6 +262,9 @@ test-docs: docs-container
 	-docker container rm -f docs-cilium >/dev/null
 	docker container run --rm -v $$(pwd):/srv/ cilium/docs-builder /bin/bash -c 'make html'
 
+install-cilium-golang-deps:
+	curl -SsL https://github.com/ksonnet/kubecfg/releases/download/v0.8.0/kubecfg-linux-amd64 > ../../../../bin/kubecfg && chmod +x ../../../../bin/kubecfg
```

Comment:
What exactly is the location of this `bin/kubecfg` directory? Why not use `GOPATH`?

Reply:
@ianvernon 

GOPATH is not required to be present

that bin directory will be `${GOPATH}/bin`

Reply:
Didn't know that, TIL. LGTM then apart from the build failure.